eng/pipelines/typespec-pr.yml

```
@@ -21,10 +21,12 @@ jobs:
     condition: succeededOrFailed()
 
   - pwsh: |
+      $verbose = ('$(System.Debug)' -eq 'true') ? "-Verbose:true" : "-Verbose:false"
```

```suggestion
      $verbose = ('$(System.Debug)' -eq 'true') ? "-Verbose" : ""
```

should we just do it this way?

---

This doesn't work either.  pwsh doesn't like empty string params.